### PR TITLE
chore: update docs for algo and hbar recoveries

### PR DESCRIPTION
**Pull request by pranavjain97**

### CODE REVIEW

File: BROADCAST_TRANSACTION.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Broadcast Transaction Instructions
+
+WRW now supports natively broadcasting transactions from the UI itself. User needs to select the coin and copy the fully signed serialized transaction. For users that use OVC (Offline Vault Console), this is the 'txHex' field from the fully signed JSON file.
+
+### This feature is currently supported for the following coins -
+- HBAR
+- ALGO (Note: BitGo doesn't have an external node for Algo and requires clients to input the details of their node)
```
Comment (alebusse) [inline BROADCAST_TRANSACTION.md:7]: This should probably say that there is no public node that can be used.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ Withdraw Funds without BitGo:
   | XLM   | Yes        | Yes  |
   | XRP   | Yes        | Yes  |
   | ZEC   | Yes        | No   |
+- | HBAR  | No         | Yes  |
+- | ALGO  | No         | Yes  |
```
Comment (alebusse) [inline README.md:30]:
```suggestion
- | HBAR  | Yes         | Yes  |
- | ALGO  | Yes        | Yes  |
```
?

File: BROADCAST_TRANSACTION.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Broadcast Transaction Instructions
+
+WRW now supports natively broadcasting transactions from the UI itself. User needs to select the coin and copy the fully signed serialized transaction. For users that use OVC (Offline Vault Console), this is the 'txHex' field from the fully signed JSON file.
```
Comment (vmccarty) [inline BROADCAST_TRANSACTION.md:3]:
```suggestion
WRW natively supports broadcasting transactions from the UI. To do so, select the coin and copy the fully-signed serialized transaction. If using the Offline Vault Console (OVC), use the 'txHex' field from the-fully signed JSON file.
```

File: BROADCAST_TRANSACTION.md (hunk 1 above)
Comment (vmccarty) [inline BROADCAST_TRANSACTION.md:7]:
```suggestion
- ALGO (Note: BitGo doesn't have an external node for Algo, so you must input your own node the details)
```

File: NON_BITGO_RECOVERY.md [hunk 4]
```diff
@@ -46,9 +46,11 @@ Build a transaction from a hot wallet using the KeyCard in order to recover fund
 
 17. For Avalanche C-Chain, the key is from https://snowtrace.io/.
 
-18. The SDK determines the amount of Coin that can be recovered, and builds a fully signed transaction to send that amount to the Destination Address.
+18. For Algorand, clients are required to use their own node to build/broadcast recovery transactions.
```
Comment (vmccarty) [inline NON_BITGO_RECOVERY.md:49]:
```suggestion
18. For Algorand, you must use your own node to broadcast recovery transactions.
```

File: NON_BITGO_RECOVERY.md [hunk 5]
```diff
@@ -46,9 +46,11 @@ Build a transaction from a hot wallet using the KeyCard in order to recover fund
...
 
-19. This completed transaction can be broadcasted on the network using a blockchain explorer to finish the recovery.
+19. The SDK determines the amount of Coin that can be recovered, and builds a fully signed transaction to send that amount to the Destination Address.
+
+20. This completed transaction can be broadcasted on the network using a blockchain explorer to finish the recovery.
```
Comment (vmccarty) [inline NON_BITGO_RECOVERY.md:53]:
```suggestion
19. The SDK determines the amount of an asset that you can recover, and then builds a fully-signed transaction to a destination address.

20. To finish the recovery, broadcast the fully-signed transaction on the network using a blockchain explorer.
```

File: UNSIGNED_SWEEP.md [hunk 6]
```diff
@@ -41,13 +41,15 @@ Build an unsigned transaction from a cold wallet using the user and backup publi
 
 15. For Avalanche C-Chain, the key is from https://snowtrace.io/.
 
-16. The SDK determines the amount of Coin that can be recovered, and builds an unsigned transaction to send that amount to the Destination Address.
+16. For Algorand, clients are required to use their own node to build/broadcast recovery transactions.
```
Comment (vmccarty) [inline UNSIGNED_SWEEP.md:44]:
```suggestion
16. For Algorand, you must use your own node to broadcast recovery transactions.
```

File: UNSIGNED_SWEEP.md [hunk 7]
```diff
@@ -41,13 +41,15 @@ Build an unsigned transaction from a cold wallet using the user and backup publi
...
 
-17. You need to half sign this unsigned transaction with your user private key using Offline Vault Console.
+17. The SDK determines the amount of Coin that can be recovered, and builds an unsigned transaction to send that amount to the Destination Address.
```
Comment (vmccarty) [inline UNSIGNED_SWEEP.md:46]:
```suggestion
17. The SDK determines the amount of an asset that you can recover, and then builds a unsigned transaction to a destination address.
```

File: UNSIGNED_SWEEP.md [hunk 8]
```diff
@@ -41,13 +41,15 @@ Build an unsigned transaction from a cold wallet using the user and backup publi
...
 
-18. You need to fully sign the half signed transaction with your backup private key using Offline Vault Console.
+18. You need to half sign this unsigned transaction with your user private key using Offline Vault Console.
 
-19. This completed transaction can be broadcasted on the network using a blockchain explorer to finish the recovery.
+19. You need to fully sign the half signed transaction with your backup private key using Offline Vault Console.
+
+20. This completed transaction can be broadcasted on the network using a blockchain explorer to finish the recovery.
```
Comment (vmccarty) [inline UNSIGNED_SWEEP.md:52]:
```suggestion
20. To finish the recovery, broadcast the fully-signed transaction on the network using a blockchain explorer.
```

File: UNSIGNED_SWEEP.md [hunk 9]
```diff
@@ -41,13 +41,15 @@ Build an unsigned transaction from a cold wallet using the user and backup publi
...
 15. For Avalanche C-Chain, the key is from https://snowtrace.io/.
 
-16. The SDK determines the amount of Coin that can be recovered, and builds an unsigned transaction to send that amount to the Destination Address.
+16. For Algorand, clients are required to use their own node to build/broadcast recovery transactions.
 
-17. You need to half sign this unsigned transaction with your user private key using Offline Vault Console.
+17. The SDK determines the amount of Coin that can be recovered, and builds an unsigned transaction to send that amount to the Destination Address.
 
-18. You need to fully sign the half signed transaction with your backup private key using Offline Vault Console.
+18. You need to half sign this unsigned transaction with your user private key using Offline Vault Console.
```
Comment (vmccarty) [inline UNSIGNED_SWEEP.md:48]:
```suggestion
18. Using the OVC (Offline Vault Console), you must sign the unsigned transaction with your private user key, resulting in a half-signed transaction.
```

File: UNSIGNED_SWEEP.md [hunk 10]
```diff
@@ -41,13 +41,15 @@ Build an unsigned transaction from a cold wallet using the user and backup publi
...
 
-19. This completed transaction can be broadcasted on the network using a blockchain explorer to finish the recovery.
+19. You need to fully sign the half signed transaction with your backup private key using Offline Vault Console.
```
Comment (vmccarty) [inline UNSIGNED_SWEEP.md:50]:
```suggestion
19. You need to fully sign the half signed transaction with your backup private key using Offline Vault Console.
```

Using the OVC again, you must sign the half-signed transaction with your private backup key, resulting in a fully-signed transaction

File: README.md (hunk 2 above)
Comment (pranavjain97) [reply README.md:30]: Thought to say Yes to prod once its tested by QA

**Comment by pranavjain97** [comment]
Thank you @vmccarty ! Those were awesome improvements, incorporated them all! 🚀

**Review by alebusse — APPROVED** [review]

**Pull request merged**